.github/workflows/pre-generate-preview-link.yml

```diff
@@ -31,5 +31,5 @@ jobs:
                   RUN_ID: ${{ github.run_id }}
               uses: actions/upload-artifact@v2
               with:
-                  name: 'pr-$RUN_ID'
+                  name: "pr-${{ env.RUN_ID }}"
```

Comment:
<img width="426" alt="image" src="https://github.com/binary-com/deriv-com/assets/56330681/a64ded40-ae2a-453f-b035-66f72c4625d4">
Should we update here too with {{ env.HEAD_REF }}

Reply:
how can i test if the current one is working or not?

Reply:
concurrency doesnt have access to env so I removed it.